openedx/features/edly/cookies.py

```diff
@@ -0,0 +1,69 @@
+from django.conf import settings
+
+from openedx.core.djangoapps.user_authn.cookies import standard_cookie_settings
+from openedx.features.edly.models import EdlySubOrganization
+from openedx.features.edly.utils import encode_edly_user_info_cookie
+
+
+def set_logged_in_edly_cookie(request, response, user):
+    """
+    Set edly user info cookie at the time of user login.
+
+    Arguments:
+        request (HttpRequest): The request to the view, used to calculate
+            the cookie's expiration date based on the session expiration date.
+        response (HttpResponse): The response on which the cookie will be set.
+        user (User): The currently logged in user.
+
+    Returns:
+        HttpResponse
+
+    """
+    if user.is_authenticated and not user.is_anonymous:
+        cookie_settings = standard_cookie_settings(request)
+        edly_cookie_string = _get_edly_cookie_string(request)
+
+        response.set_cookie(
+            settings.EDLY_USER_INFO_COOKIE_NAME.encode('utf-8'),
+            edly_cookie_string,
+            **cookie_settings
+        )
+
+    return response
+
+
+def delete_logged_in_edly_cookie(response):
+    """
+    Delete edly user info cookie.
+    Arguments:
+        response (HttpResponse): The response sent to the client.
+    Returns:
+        HttpResponse
+    """
+    response.delete_cookie(
+        settings.EDLY_USER_INFO_COOKIE_NAME.encode('utf-8'),
+        path='/',
+        domain=settings.SESSION_COOKIE_DOMAIN
+    )
+
+    return response
+
+def _get_edly_cookie_string(request):
+    """
+    Gets data to put in edly user info cookie and encodes it through JWT.
+    Arguments:
+        request (HttpRequest): The request to the view, used to get site
+        object to get relevant edly organization and edly sub organization.
+    Returns:
+        string
+    """
+    try:
+        edly_sub_organizations = EdlySubOrganization.objects.get(lms_site=request.site)
```

Comment:
use reverse relation from request.site.edlysuborganization ....

Reply:
I tried doing this but couldn't get any reverse related object from the site object. It doesn't seem to be appearing in the site object as well. I will still verify it again.
![image](https://user-images.githubusercontent.com/20854114/81265845-2ed09000-905d-11ea-8ca2-eeb1786007d8.png)

Reply:
@rehan99000 we have relationship setup of `EdlySubOrganization` with `lms_site` and `studio_site` 
https://github.com/edly-io/edx-platform/pull/71/files